pages/partners/farmlands.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <main>
+    <div
+      class="bg-cover bg-[#174B33] bg-blend-overlay bg-[url('~/assets/images/countryside.png')]"
+    >
+      <div class="max-w-6xl mx-auto">
+        <div class="text-content-on-color text-base leading-6 uppercase font-semibold tracking-wider pt-16 md:pt-32 pl-4 md:pl-40 pb-3">
+          Project
+        </div>
+        <div class="text-content-inverse-primary text-4xl leading-10 font-extrabold px-4 md:px-40 py-3">
+          Farmlands
+        </div>
+        <div class="text-content-inverse-secondary text-xl leading-7 font-normal px-4 md:pl-40 md:pr-48 py-3">
+          Farmlands has partnered with Centrapay to provide new ways for your business to accept Farmlands Cards faster, easier and more securely than ever.
+        </div>
+        <div class="text-content-inverse-secondary type-subtitle-2 px-4 md:pl-40 pt-3 pb-0.5">
+          Made By
+        </div>
+        <div class="flex items-center px-4 md:pl-40 pt-0.5 pb-3">
+          <div class="text-content-inverse-primary pr-2">
+            Centrapay
+          </div>
+          <icons-verified-check class="w-4 h-4" />
+        </div>
+        <div class="text-content-inverse-secondary type-subtitle-2 px-4 md:pl-40 pt-3 pb-0.5">
```

Comment:
The same classes (`md:pl-40 pt-3 pb-0.5`) are repeating here so there may be a better way to structure this inside another div that has `md:pl-40 space-y-3` or similar.